### DISCUSSION TITLE
my canvas keeps crashing

### DISCUSSION THREAD

**Discussion by ithalvrnz** [thatskymod/Sky-CotL-Scripts#62] [Q&A]
i tried removing each libs i use but it's still not working because canvas still crashed every time my loading screen almost finished loading

**Comment by pentapapers** [comment]
I have the same problem. For me, it works the first time I start the app after installing. But the next time, it crashes.

Also, Langit crashes when I try to return to Home space and then the same problem happens whenever I try to reenter the app.

**Comment by thatskymod** [comment]
This seems odd given that you've removed any libs that you use.

Could you specify your Canvas version, Sky build number and device model?
